src/redux/features/plantings/plantingsSelectors.ts

```diff
@@ -72,5 +73,34 @@ export const selectPlantingProgressSubzones = createSelector(
 );
 
 // selector for single site
-export const getReportedPlantsForSite = (state: RootState, siteId: number) =>
-  selectPlantingProgressSubzones(state)?.find((report) => report.siteId === siteId);
+export const selectPlantingProgressForSite = (state: RootState, siteId: number) =>
```

Comment:
@charlie-tf I renamed this selector name for consistency in case you are already using this for maps (the intended use-case)

Reply:
hmm, i was working on this and instead was using the populations selector (https://github.com/terraware/terraware-web/blob/main/src/redux/features/tracking/sitePopulationSelector.ts) to get the data for the subzones on the map. Seems like there's some redundancy going on. Not sure quite how to control for it...

Reply:
The populations have species information, which the map needs

Reply:
But not "planting complete" so I'll have to interact with these selectors anyways, it would seem

Reply:
Oh that's totally fine. we can remove this selector then. Will update the PR.

Reply:
The presentation of map versus table is quite different for sure. Updated the PR, removed the unwanted selector.

Reply:
> But not "planting complete" so I'll have to interact with these selectors anyways, it would seem

If you want planting completed information alone, you can get that from the subzone (selectPlantingSites selector), don't need to go via this route. Maybe add a selector that combines the populations view + plantingCompleted status

Reply:
Ah, that's right 👍

Reply:
yep :) here the plantingCompleted is fetched from planting sites data as well